target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -218,6 +218,15 @@ tplink,cpe510-v3)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "tp-link:green:link3" "wlan0" "51" "100" "-50" "13"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "tp-link:green:link4" "wlan0" "76" "100" "-75" "13"
 	;;
+tplink,cpe220-v3)
+	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth1"
```

Comment:
Any reason why you do not label this "lan0" and merge it with definitions for cpe210-v1?